src/data/__mocks__/items.js

```diff
@@ -140,6 +140,19 @@ export const sampleCropItem3 = testItem({
   },
 })
 
+export const sampleCropItem4 = testItem({
+  cropType: 'SAMPLE_CROP_TYPE_4',
+  doesPriceFluctuate: true,
+  id: 'sample-crop-4',
+  name: 'Sample Crop Item 4',
+  type: itemType.CROP,
+  value: 2,
+  cropTimetable: {
+    [SEED]: 2,
+    [GROWING]: [1, 2, 1],
+  },
+})
+
```

Comment:
I'd like us to get away from using the `sampleCropItem`s for tests. This was one of my less-good ideas from early on in the project's development that there are better solutions for. It seems that `carrot` should cover our needs for testing growth phases, so let's use that for the tests introduced in this PR.

Reply:
will do! i didn't actually realize you wanted to get away from these, so i kinda just followed the existing pattern. i fully support using the actual crops here though

Reply:
i went ahead and updated all the tests using mocked items in this test file